.github/workflows/ci.yml

```diff
@@ -24,7 +23,12 @@ jobs:
         run: sudo apt-get update
 
       - name: Install tornettools dependencies
-        run: sudo apt install -y python3 python3-dev python3-pip libxml2 libxml2-dev libxslt1.1 libxslt1-dev libpng16-16 libpng-dev libfreetype6 libfreetype6-dev libblas-dev liblapack-dev
+        run: sudo apt-get install -y python3 python3-dev python3-pip libxml2 libxml2-dev libxslt1.1 libxslt1-dev libpng16-16 libpng-dev libfreetype6 libfreetype6-dev libblas-dev liblapack-dev
```

Comment:
maybe separate the packages so that they appear on one per line (for readability and to make later diffs easier).

Reply:
Added